src/TestSuite/Constraint/Response/BodyContains.php

```diff
@@ -30,7 +30,7 @@ class BodyContains extends ResponseBase
     /**
      * Constructor.
      *
-     * @param Response $response Response
+     * @param \Cake\Http\Response $response \Cake\Http\Response
```

Comment:
Dup error

Reply:
Seems the sniffer is a bit overzealous and modifies descriptions too. I will open an issue with slevomat standards.

Reply:
slevomat/coding-standard#394